reactor-core/src/main/java/reactor/core/publisher/MonoSource.java

```diff
@@ -86,6 +89,9 @@ public Object scanUnsafe(Attr key) {
 		if (key == Attr.PARENT) {
 				return source;
 		}
+		if (key == RUN_STYLE) {
+		    return SYNC;
```

Comment:
isn't the same as https://github.com/reactor/reactor-core/pull/2123#discussion_r427904157 ?

Reply:
Fixed in commit [a69fc2a](https://github.com/reactor/reactor-core/pull/2123/commits/a69fc2ac7f60a06b480ab1273d97afaa47cfc036)